pom.xml

```diff
@@ -1871,6 +1876,16 @@
                 <artifactId>org.wso2.carbon.identity.cors.mgt.server.feature</artifactId>
                 <version>${carbon.identity.framework.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.framework</groupId>
+                <artifactId>org.wso2.carbon.identity.user.functionality.mgt.server.feature</artifactId>
+                <version>${carbon.identity.framework.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.framework</groupId>
```

Comment:
Do we need this as well or having the server feature is sufficient?

Reply:
Resolved with https://github.com/wso2/product-is/pull/8502/commits/4193975e6aed6857f6efb4fe502d88b91e25bd72.